todo/src/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import MainContainer from './components/MainContainer';
+import reducer from './store/reducers/reducer';
+
+const store = createStore(reducer);
+
+ReactDOM.render(
+    <Provider store={store}>
+        <MainContainer />
```

Comment:
I would leave this one to App. Although it won't have any breaks, you have to make sure other people can follow your structure.